src/main/java/decodex/commands/DataCommand.java

```diff
@@ -0,0 +1,30 @@
+package decodex.commands;
+
+import decodex.data.Data;
+import decodex.data.DataManager;
+import decodex.data.exception.CommandException;
+import decodex.modules.ModuleManager;
+import decodex.ui.Ui;
+
+public class DataCommand extends Command {
+
+    public static final String COMMAND_WORD = "data";
+
+    private final String argument;
+
+    public DataCommand(String argument) {
+        super();
+        this.argument = argument;
+    }
+
+    @Override
+    public void run(DataManager dataManager, ModuleManager moduleManager, Ui ui) throws CommandException {
+        if (argument.isEmpty()) {
+            throw new CommandException("[-] Missing argument");
+        }
+        String dataString = argument;

```

Comment:
Since `dataString` takes `argument`'s value, and both variables have the same purpose, this line can be omitted if `dataString` is a class variable.